### DISCUSSION TITLE
the post is not publishing in the page

### DISCUSSION THREAD

**Discussion by AxelCotonGutierrez** [community/community#68842] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a mathematical blog on GitHub where I keep adding games in JavaScript, CSS, and HTML. Everything is working fine, but today I tried to upload a post that contains a small JavaScript test. It uploaded successfully to the repository, but it doesn't appear on the page. Does anyone know why?

Page: [https://axelcotongutierrez.github.io/learningmathematicas/](url)

Post in Respository: [https://github.com/AxelCotonGutierrez/learningmathematicas/blob/master/_posts/2023-09-30-TSND.html](url)

**Comment by vishalkrishna8** [comment]
Hi @AxelCotonGutierrez , Check the file and folder structure, correct case sensitivity, file extensions, and HTML references. Ensure your browser cache is cleared, and your GitHub repository is up to date.

**Comment by seyLu** [comment, marked as answer]
#### Hi @AxelCotonGutierrez,

Blog posts don't get seen by the public unless it is on or before the published date.

![image](https://github.com/community/community/assets/98249191/8cf39d1b-2c77-49b9-b78c-1b75c4fa34d0)

![image](https://github.com/community/community/assets/98249191/04474675-017e-4cb3-a358-ac13320fff7e)

Just redeploy the site and the post should be up.

**Reply by AxelCotonGutierrez** [reply]
Thanks a lot
